robottelo/common/constants.py

```diff
@@ -236,6 +250,7 @@
 FAKE_3_PUPPET_REPO = "http://omaciel.fedorapeople.org/fakepuppet03"
 FAKE_4_PUPPET_REPO = "http://omaciel.fedorapeople.org/fakepuppet04"
 FAKE_5_PUPPET_REPO = "http://omaciel.fedorapeople.org/fakepuppet05"
+FAKE_6_PUPPET_REPO = "http://kbidarka.fedorapeople.org/repos/puppet-modules/"
```

Comment:
why another puppet repo?
This puppet repo contains a single puppet-module called 'motd', the idea is to sync, publish, promote as little content as possible to save time. 
Also the
a) directory structure required to sync puppet module is fine and
b) we do have modules.json file for the sync to be successful.